src/servers/src/http.rs

```diff
@@ -743,6 +743,7 @@ impl HttpServer {
                 "/pipelines/:pipeline_name",
                 routing::delete(event::delete_pipeline),
             )
+            .route("/test_pipelines", routing::post(event::test_pipeline))
```

Comment:
Maybe something like `"/pipelines/:pipeline_name/simulate"` or `"/logs/dryrun"` is more formal.